src/api/offers.js

```diff
@@ -16,23 +17,31 @@ const validateMiddleware = (req, res, next) => {
 export default ({offerService}) => {
   const router = new Router()
 
-  router.get('/getAll', (req, res) => {
-    res.json(offerService.getAllOffers())
+  router.get('/getAll', async (req, res) => {
+    try {
+      const queryResult = await offerService.getAllOffers()
+      return res.json(queryResult)
+    } catch (e) {
+      return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json(e)
+    }
   })
 
-  router.get('/', (req, res) => {
-    res.json(offerService
-      .getActiveOffers(req.city)
-      .map(offer => offer.formData))
+  router.get('/', async (req, res) => {
+    try {
+      const queryResult = await offerService.getActiveOffers(req.city)
+      return res.json(queryResult)
+    } catch (e) {
+      return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json(e)
+    }
   })
 
   router.put('/', [
     body('email').isEmail().trim().normalizeEmail(),
     body('duration').isInt().toInt().custom(value => [3, 7, 14, 30].includes(value)),
+    body('formData').exists(),
     validateMiddleware,
     async (req, res) => {
```

Comment:
I'd propose to write `request` and `result` always.

Reply:
Yes, that's probably better. Can you do it in this task @NKlug ? :D

Reply:
I'll do it.